testing/application/src/My/Controller/MyController.php

```diff
@@ -1,9 +1,13 @@
 <?php
+
 namespace My\Controller;
+
 use Symfony\Component\HttpFoundation\Response;
 
-class MyController {
-    public function route1() {
-       return new Response('<html><body>Hello world</body></html>', 200);
+class MyController
```

Comment:
If this class is only for testing, it has no purpose of being extended by some project or other libraries, even ourself. It could safely became final.

```suggestion
final class MyController
```